docs/konnector_declaration_v3.md

```diff
@@ -28,6 +28,7 @@ remarks are welcome.
 {
   "version": "1.0.0",
     "name": "Trainline",
+    "type": "node",
```

Comment:
Actually, I supposed the ```main``` section would be in the package.json but that way, we would not have to read the package.json at all and we would run the konnector with a simple node command not using npm. Ok, I put the main section in the manifest.